docs/loading/S3_Load.md

```diff
@@ -0,0 +1,272 @@
+# Load data from Amazon S3
+
+StarRocks provides two options for loading data from S3:
+
+1. Asynchronous loading using Broker Load (`WITH BROKER`)
+2. Synchronous loading using the `FILES()` table function
+
+Small datasets are often loaded synchronously using the `FILES()` table function, and large datasets are often loaded asynchronously using Broker Load. The two methods have different advantages and are described below.
+
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/8d5afd51-e8bb-40b2-92c7-0a854c5f47a7)
Is this part an example of the common content you talked about in the "Doc team" group? And the part will be included in a separate file so it could be added anywhere as needed?

Reply:
Yes! I moved that into an include file yesterday and will include it. Thanks for the reminder!